os-eap-txnrecovery/bash/added/partitionPV.sh

```diff
@@ -174,6 +174,11 @@ function migratePV() {
     done
   fi
 
+  # server can only be configured once due to CLI script being executed
+  # doing it before the loop that will start a server per directory.
+  source $JBOSS_HOME/bin/launch/launch.sh
```

Comment:
would not be more readable (if possible) to move the `source .../launch.sh` (which probably only imports some functions) to the top of the file where all other `source` calls are placed?

Reply:
@ochaloup , partitionPV.sh is also sourced from openshift-launch.sh, sourcing launch.sh has a side effect (creates an execution context) that shouldn't be created when partitionPV.sh is sourced in openshift-launch.sh

Reply:
oh, I see. please discard my comment then :)